sark/codeblock.py

```diff
@@ -61,7 +61,7 @@ def __repr__(self):
 
 
 class FlowChart(idaapi.FlowChart):
-    def __init__(self, f=None, bounds=None, flags=0):
+    def __init__(self, f=None, bounds=None, flags=ida_gdl.FC_PREDS):
```

Comment:
`ida_gdl` is not currently imported. Also, I'd prefer to use `idaapi` here to maintain compatibility with older IDA versions.